tests/test_hashing.py

```diff
@@ -0,0 +1,110 @@
+import manim.utils.hashing as hashing
+
+import json
+
+
+def test_JSON_basic():
+    o = {"test": 1, 2: 4, 3: 2.0}
+    o_serialized = hashing.get_json(o)
+    assert isinstance(o_serialized, str)
+    assert o_serialized == str({"test": 1, "2": 4, "3": 2.0}).replace("'", '"')
+
+
+def test_JSON_with_object():
+    class Obj:
+        def __init__(self, a, b):
+            self.a = 2
+            self.b = 3.0
+            self.c = [1, 2, "test", ["nested list"]]
+            self.d = {2: 3, "2": "salut"}
+
+    o = Obj(1, 2)
```

Comment:
Nitpick: the init arguments `a` and `b` are not used. Was this intentional?

Reply:
Yes, my mistake. Thanks !